src/test/java/api/requests/scenarios/MoveRequestTests.java

```diff
@@ -697,6 +697,80 @@ public void canMoveARecallRequestAsHoldRequest()
     assertThat(interestingTimesQueue.getTotalRecords(), is(2));
   }
 
+  @Test
+  public void cannotDisplacePagedRequest()
+      throws MalformedURLException,
+      InterruptedException,
+      TimeoutException,
+      ExecutionException {
+
+      final IndividualResource secondFloorEconomics = locationsFixture.secondFloorEconomics();
+      final IndividualResource mezzanineDisplayCase = locationsFixture.mezzanineDisplayCase();
+
+      final IndividualResource itemCopyA = itemsFixture.basedUponTemeraire(
+        holdingBuilder -> holdingBuilder
+          .withPermanentLocation(secondFloorEconomics)
+          .withNoTemporaryLocation(),
+        itemBuilder -> itemBuilder
+          .withNoPermanentLocation()
+          .withNoTemporaryLocation()
+          .withBarcode("10203040506"));
+
+      final IndividualResource itemCopyB = itemsFixture.basedUponTemeraire(
+        holdingBuilder -> holdingBuilder
+          .withPermanentLocation(mezzanineDisplayCase)
+          .withNoTemporaryLocation(),
+        itemBuilder -> itemBuilder
+          .withNoPermanentLocation()
+          .withNoTemporaryLocation()
+          .withBarcode("90806050402"));
+
+    IndividualResource james = usersFixture.james();
+    IndividualResource steve = usersFixture.steve();
+    IndividualResource jessica = usersFixture.jessica();
+
+    // James Checks out Item Copy A
+    loansFixture.checkOutByBarcode(itemCopyA, james, DateTime.now(DateTimeZone.UTC));
+
+    // Steve requests Item Copy B
```

Comment:
Did you mean Item Copy **A** in the comment because in the next lines Steve actually requested Item Copy A?

Reply:
I will update the comment